### PR TITLE
Amtool

### PR DESCRIPTION
**Pull request by zloeber**
- Adds amtool per #149

### CODE REVIEW

**Comment by osterman** [comment]
@zloeber build is failing:
```
/bin/bash: /codefresh/volume/packages/tmp/build.mjnPAk/src/amtool: cannot execute binary file: Exec format e
rror   
```

This is usally because package depends on `libc6-compat`

To debug, this is what I would run:
```
make docker/build/apk/shell
make -C vendor/amtool apk
```

**Comment by zloeber** [comment]
Sorry, I'm looking into this and the build error is because the test of the executable is actually trying to run the tar.gz file for reasons I've yet to discern. I made some minor changes to fall more in line with what other packages are doing for instances where the repo name does not align with the app being installed but cannot seem to get the build to test properly regardless. I suppose the closest model app would be fzf (in theory at least). I'll keep digging into this when I can but will take any more hints you have to give :)

**Comment by osterman** [comment]
Aha, ok I will take a look. Don’t spend too much time ;)

Maybe look at where we define a “prepare*” target

I ran into this the other day, but forget which package.

**Comment by zloeber** [comment]
I believe that the auto-label tasks are failing the build at this point. I haven't quite figured out the auto-label goals yet but I get make errors about the use of : and :: on the same task (I worked around the local build issues by changing the tasks but wasn't certain what else would be broken upstream because of it). In another pull request of more questionable value (feature/package_helpers) I left the makefile changes for auto-label though.

Thanks

**Comment by osterman** [comment]
>  I haven't quite figured out the auto-label goals yet but I get make errors about the use of : and :: on the same tas

re-run `make init` as this was fixed upstream in the `build-harness` https://github.com/cloudposse/build-harness/pull/187

**Comment by osterman** [comment]
Ignore all the `auto-*` failures - those won't run on forks due to github security model

File: README.md [hunk 1]
```diff
@@ -477,7 +491,6 @@ Check out [our other projects][github], [follow us on twitter][twitter], [apply
   [email]: https://cpco.io/email?utm_source=github&utm_medium=readme&utm_campaign=cloudposse/packages&utm_content=email
   [commercial_support]: https://cpco.io/commercial-support?utm_source=github&utm_medium=readme&utm_campaign=cloudposse/packages&utm_content=commercial_support
   [we_love_open_source]: https://cpco.io/we-love-open-source?utm_source=github&utm_medium=readme&utm_campaign=cloudposse/packages&utm_content=we_love_open_source
-  [terraform_modules]: https://cpco.io/terraform-modules?utm_source=github&utm_medium=readme&utm_campaign=cloudposse/packages&utm_content=terraform_modules
```
Comment (osterman) [inline README.md]: Deleting this line is failig the linting process. Did you delete it manually? The `README.md` is generated dynamically.

**Comment by osterman** [comment]
Just one thing... The `README.md` is generated from the `README.yaml`. Make all your changes to the `README.yaml`, then rebuild. 

Please rebuild the `README.md` by running the following commands:
```
make init
make readme/deps
make readme
```

There might be a lot of changes in the `README.md` as a result. That's okay! 

Sorry for the inconvenience!

File: README.md (hunk 1 above)
Comment (zloeber) [reply README.md]: Honestly not certain how that happened. Sorry I'm having such a hard time with this one.

**Pull request merged**